src/popup/components/TransactionItem.vue

```diff
@@ -53,21 +53,24 @@
 
 <script lang="ts">
 import { SCHEMA } from '@aeternity/aepp-sdk';
-import { computed, defineComponent, PropType } from '@vue/composition-api';
+import {
+  computed, defineComponent, PropType,
```

Comment:
If ESLint suggest to break the import into separate lines I think it's better to move every item into separate line.